x-pack/legacy/plugins/infra/public/components/metrics_explorer/chart.tsx

```diff
@@ -18,8 +18,6 @@ import {
 } from '@elastic/charts';
```

Comment:
Most of the changes in this file look like indentation, for the most part, right?

Reply:
Yeah, it's the same for any file where `injectUICapabilities` was removed.